content/release-10-14-0/applications-10-14-0-bundle/10-14-0-402.md

```diff
@@ -0,0 +1,31 @@
+---
+weight: 12
+title: Release 10.14.0.402
+layout: bundle
+---
+
+<!--10.14.0.394 - 10.14.0.402-->
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Web SDK</td>
+<td>In the data-grid component, when filtering and using the enter key, there was a delay in some specific cases when using additional input method editors, for example for Japanese. This is now fixed and filtering works as expected.</td>

```

Comment:
```suggestion
<td>In the data-grid component, when filtering and using the enter key, there was a delay in some specific cases when using additional input method editors, for example, for Japanese. This is now fixed and filtering works as expected.</td>
```
what is an input method editor?

Reply:
Originally it said IMP, which I looked up and found that this stands for input method editor. Should I better capitalize it?